exercises/say/say_test.rb

```diff
@@ -1,128 +1,116 @@
-#!/usr/bin/env ruby
-gem 'minitest', '>= 5.0.0'
 require 'minitest/autorun'
 require_relative 'say'
 
+# Test data version: e3bd4a2
 class SayTest < Minitest::Test
-  def test_0
-    assert_equal 'zero', Say.new(0).in_english
+  def test_zero
+    # skip
+    question = '0'
+    assert_equal(zero, SayProblem.new(question).answer)
   end
 
   def test_one
     skip
-    assert_equal 'one', Say.new(1).in_english
+    question = '1'
+    assert_equal(one, SayProblem.new(question).answer)
   end
 
-  def test_14
+  def test_fourteen
     skip
-    assert_equal 'fourteen', Say.new(14).in_english
+    question = '14'
+    assert_equal(fourteen, SayProblem.new(question).answer)
   end
 
   def test_twenty
     skip
-    # This really shouldn't be twenty-zero
-    assert_equal 'twenty', Say.new(20).in_english
+    question = '20'
+    assert_equal(twenty, SayProblem.new(question).answer)
   end
 
-  def test_twenty_two
+  def test_twenty-two
```

Comment:
Hi there! Looks like the build is failing because this method (and a few others) have invalid names.

`test_twenty-two` rather than `test_twenty_two`